### DISCUSSION TITLE
Snapshot or historical view

### DISCUSSION THREAD

**Discussion by mharward-gr** [Unleash/unleash#8577] [Q&A]
Is there some way to find out a snapshot or historical view of feature toggles at a specific datetime in the past? Probably in the context of an environment, though wider would still be useful.

**Comment by chriswk** [comment]
There's no built in support for this, but I guess you could use the events log. Read events up to a certain time to get something akin to a snapshot state.

I can see the use of a feature like this, but it is not something that has been prioritized thus far.
